epub34/rs/index.html

```diff
@@ -2325,8 +2322,8 @@ <h2>Accessibility</h2>
 				<li>Document Object Model (DOM) &#8212; Provide access to the [[dom]] of EPUB content documents so that
 					users can investigate the semantics and structure expressed in the source.</li>
 				<li>Table of Contents &#8212; Ensure that users can access the link text, including <a
-						data-cite="epub-34#confreq-nav-a-title">text alternatives for visual content</a>. Activating the
-					table of contents links should be possible using different inputs.</li>
+						data-cite="epub-34#confreq-nav-a-title">text alternatives for visual content</a>. Ensure users
+					can activate the table of contents links using different inputs.</li>
```

Comment:
```suggestion
						data-cite="epub-34#confreq-nav-a-title">text alternatives for visual content</a>, and that users
					can activate the table of contents links using different inputs.</li>
```

(It avoids repeating the word "Ensure" in two consecutive sentences...)